src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2136,12 +2135,13 @@ private static IEnumerable<Diagnostic> FilterDiagnosticsByLocation(IEnumerable<D
             }
         }
 
-        internal ImmutableArray<Diagnostic> GetDiagnosticsForSyntaxTree(
+        internal override ImmutableArray<Diagnostic> GetDiagnosticsForSyntaxTree(
             CompilationStage stage,
             SyntaxTree syntaxTree,
             TextSpan? filterSpanWithinTree,
             bool includeEarlierStages,
-            CancellationToken cancellationToken = default(CancellationToken))
+            bool includeDiagnosticsWithSourceSuppression,
```

Comment:
Actually, the design team proposed that we remove this flag from GetXXXDiagnostics APIs and make it a CompilationOption. So the changes for threading this flag will be reverted in next PR.